vsintegration/src/FSharp.Editor/Common/Pervasive.fs

```diff
@@ -20,6 +23,27 @@ module String =
                 yield String.Empty
         |]
 
+let (</>) path1 path2 = Path.Combine(path1,path2)
+
+type Path with
+    static member GetFullPathSafe path =
+        try Path.GetFullPath path
+        with _ -> path
+
+    static member GetFileNameSafe path =
+        try Path.GetFileName path
+        with _ -> path
+
+
+/// Load times used to reset type checking properly on script/project load/unload. It just has to be unique for each project load/reload.
+/// Not yet sure if this works for scripts.
+let fakeDateTimeRepresentingTimeLoaded x = DateTime(abs (int64 (match x with null -> 0 | _ -> x.GetHashCode())) % 103231L)
```

Comment:
Why is this added in this PR?  It doesn't seem to be mentioned elsewhere in the PR?